cheroot/_compat.py

```diff
@@ -14,6 +14,14 @@
 except ImportError:
     import selectors2 as selectors  # noqa: F401  # lgtm [py/unused-import]
 
+try:
+    import socket  # noqa: F401  # lgtm [py/unused-import]
```

Comment:
It's better to have just one instruction in try/except block + this import triggers two LGTM violations, not just one:
```suggestion
import socket  # noqa: F401  # lgtm [py/unused-import] lgtm [py/import-and-import-from]
try:
```

Reply:
i am not totally sure on this - i *think* it may be needed because `backports.socketpair` modifies the `socket` module, and those modifications wouldn't be visible outside that module unless we imported it here first, but i am not familiar enough w the python import mechanism, and don't have an affected platform to test on directly. suggestions welcome :)

Reply:
Have you tested whether it's different with this line removed?

Reply:
not yet - i don't have an affected platform to test on, and the platforms in ci that would exercise that code were unresponsive prior to rebasing to get some of those ssl fixes. i can push a couple experiments.

Reply:
Ah, so only did it because of the example in their readme? I'm pretty sure that it's unnecessary.